csm_web/frontend/src/components/MentorSection.js

```diff
@@ -247,7 +259,132 @@ class MentorSectionAttendance extends React.Component {
   }
 }
 
+function zeroPadTwoDigit(num) {
+  return num < 10 ? `0${num}` : num;
+}
+
+class SpacetimeEditModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { location: "", day: "Mon", time: "", isPermanent: false, changeDate: "", showSaveSpinner: false };
```

Comment:
Default `day` to Monday because that's the value that's selected by default when the modal opens, so it would be very confusing for request to fail because the user did not _explicitly_ select Monday.

Reply:
might want to make a constant called `DEFAULT_DAY` or similar, rather than hardcoding it

Reply:
Actually just added an un-selectable but default shown placeholder option (with text shown as "---") instead of doing this.